lib/ui/screen/root_navigation_bar.dart

```diff
@@ -1,27 +1,24 @@
 import 'package:conference_2023/l10n/localization.dart';
-import 'package:conference_2023/ui/router/router_app.dart';
 import 'package:conference_2023/ui/screen/root_tab.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
 
 class RootNavigationBar extends ConsumerWidget {
   const RootNavigationBar({
     super.key,
-    required this.currentTab,
+    required this.navigationShell,
   });
 
-  final RootTab currentTab;
+  final StatefulNavigationShell navigationShell;
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final localization = ref.watch(localizationProvider);
 
-    final (selectedIndex, indicatorColor) = switch (currentTab) {
-      RootTab.home => (0, null),
-      RootTab.sessions => (1, null),
-      RootTab.venue => (2, null),
-      _ => (0, Colors.transparent),
-    };
+    final (selectedIndex, indicatorColor) = navigationShell.currentIndex > 2
```

Comment:
Could you please reverse the bool value?
Here I would like to determine as a case where branchIndex is less than or equal to 2 and not.

Reply:
Sure, reflected.
[chore: change bool value condition](https://github.com/FlutterKaigi/conference-app-2023/pull/142/commits/27e57168ac30aca9f29ea94e06e3fdd50ece0915)